ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -44,7 +44,7 @@ static bool within_tolerance (double val1, double val2, double tolerance)
 
 static void (* CommandStatusCallback) (int,bool);
 
-const double PanTiltTimeout = 5.0; // seconds, made up
+const double PanTiltTimeout = 15.0; // seconds, made up
```

Comment:
This seems a large timeout value for a pan tilt operation, is this justified?

Reply:
I believe so, for reference going from 0 - 70 degrees takes just under 6 seconds. 15 was the smallest value that I found in testing that still allowed for the largest possible change (180).

Reply:
I'm fine with this timeout -- thanks for the experimentation, @Keeganfn !